workspace/user.go

```diff
@@ -81,9 +81,12 @@ func ListRoles(workspaceID string, client *houston.Client, out io.Writer) error
 	for i := range workspace.RoleBindings {
 		role := workspace.RoleBindings[i]
 		var color bool
-		tab.AddRow([]string{role.User.Username, role.User.ID, role.Role}, color)
+		if role.User.Username != "" {
+			tab.AddRow([]string{role.User.Username, role.User.ID, role.Role}, color)
+		} else {
```

Comment:
since `ServiceAccount` is a pointer, might want to check if its not `nil` before dereferencing
```suggestion
		} else if role.ServiceAccount != nil {
```
else you can also change the type of Service to `WorkspaceServiceAccount` unless its required to be a pointer